config/prod/sara-working-machine.yaml

```diff
@@ -1,6 +1,6 @@
 # Provision an EC2 instance
 template_path: remote/managed-ec2-ubuntu-v1.yaml
```

Comment:
there's a new version, https://github.com/Sage-Bionetworks/sandbox-provisioner/pull/121/files, you'll want to use that one.